app/src/test/java/org/simple/clinic/home/overdue/OverdueUpdateTest.kt

```diff
@@ -363,4 +363,18 @@ class OverdueUpdateTest {
             hasEffects(ToggleOverdueAppointmentSelection(appointmentId))
         ))
   }
+
+  @Test
+  fun `when select all button is clicked, then add all appointments Ids`() {
```

Comment:
Maybe we can rename this test to - `then select all appointment ids`? As in this test name its not very clear on where are we adding these IDs to